.travis.yml

```diff
@@ -23,19 +23,24 @@ stages:
 jobs:
   allow_failures:
     - script: ./script/docker-build-and-push
+    - script: "bundle exec rake update_static_files"
   include:
     - stage: "test"
       sudo: required
       dist: trusty
       env:
       script: ./script/validate-example-payloads-with-docker
+    - stage: "test update_static_files Rake task"
+      script: "bundle exec rake update_static_files"
     - stage: ":ship: it to Quay.io"
       sudo: required
       dist: trusty
       env:
       script: ./script/docker-build-and-push
 
-script: bundle exec rake && ./script/validate-bash-syntax
+script:
+  - mkdir -p public/files && touch public/files/nvm.sh
+  - bundle exec rake lib/travis/build/addons/sauce_connect/sauce_connect.sh spec && ./script/validate-bash-syntax
```

Comment:
This Rake task generates `sauce_connect.sh`, which a spec relies on. See https://travis-ci.org/travis-ci/travis-build/jobs/258148161#L575 for a failure.